pages/instructorAssessments/instructorAssessments.ejs

```diff
@@ -9,6 +9,7 @@
   <body>
     <%- include('../partials/navbar'); %>
     <div id="content" class="container-fluid">
+      <%- include('../partials/courseInstanceSyncErrorsAndWarnings'); %>
```

Comment:
If I understand correctly, this partial should only be included on pages that have a single `locals.assessment`? This page has many assessments, so this partials code won't do anything?

Reply:
Note that this is `courseInstanceSyncErrorsAndWarnings`, not `assessmentSyncErrorsAndWarnings` - there's actually four different `*SyncErrorsAndWarnings.ejs` partials.